src/utils/redux/slices/multisigsSlice.ts

```diff
@@ -62,6 +62,19 @@ export const multisigsSlice = createSlice({
         account.label = newName;
       }
     },
+    removeMultisigs: (state, { payload }: { payload: RawPkh[] }) => {
```

Comment:
this would look much cleaner:
```typescript
removeMultisigs: (state, { payload: addresses }: { payload: RawPkh[] }) => {
      const operationIdsToRemove = state.items
        .filter(multisig => addresses.includes(multisig.address.pkh))
        .map(multisig => String(multisig.pendingOperationsBigmapId));

      state.pendingOperations = omit(state.pendingOperations, operationIdsToRemove);
      state.labelsMap = omit(state.labelsMap, addresses);
    },
```